lighthouse-core/report/report-generator.js

```diff
@@ -214,7 +214,20 @@ class ReportGenerator {
    * @return {!Array<string>} an array of CSS
    */
   getReportCSS() {
-    return [fs.readFileSync(path.join(__dirname, './styles/report.css'), 'utf8')];
+    // Cannot DRY this up and dynamically create paths because fs.readdirSync
+    // doesn't browserify well with a variable path. See https://github.com/substack/brfs/issues/36.
+    const partialStyles = [
+      fs.readFileSync(path.join(__dirname, '../formatters/partials',
```

Comment:
nit: can just do `__dirname + '../formatters/partials/critical-request-chains.css'` (for all of these)